src/webpackPlugins/CustomTasksPlugin.js

```diff
@@ -47,9 +47,9 @@ class CustomTasksPlugin {
      * Minify the given asset file.
      */
     minifyAssets() {
-        let tasks = Mix.tasks.filter(
-            task => task.constructor.name !== 'VersionFilesTask'
-        );
+        let tasks = Mix.tasks.filter(task => {
+            return task.constructor.name !== 'VersionFilesTask' && task.constructor.name !== 'CopyFilesTask';
```

Comment:
i simpler way for this this check:

```js
let tasks = Mix.tasks.filter(
    task => ! ['CopyFilesTask', 'VersionFilesTask'].includes(task.constructor.name)
);
```

or just add the new check without changing the ecma:

```js
let tasks = Mix.tasks.filter(
    task => task.constructor.name !== 'VersionFilesTask' && task.constructor.name !== 'CopyFilesTask'
});
```

Reply:
@dbpolito The top one is nice. Love that.